ensimesublime/protocol.py

```diff
@@ -238,7 +239,13 @@ def apply_refactor(self, call_id, payload):
             self.env.logger.warning("Couldn't parse diff_file: {}"
                                     .format(diff_file))
             return
-        result = patch_set.apply(0, "/")
+        path =  Path(self.refactorings[payload['procedureId']])
```

Comment:
Using `PurePath` would be more suitable as it can't access the os and we don't want it to anyway. What do you think?